controllers/openstackprovisionserver_controller.go

```diff
@@ -252,12 +231,12 @@ func (r *OpenStackProvisionServerReconciler) Reconcile(ctx context.Context, req
 		if podListLen > 1 {
 			common.LogForObject(r, fmt.Sprintf("WARNING: Multiple pods (%d) found for OpenStackProvisionServer %s!", podListLen, instance.Name), instance)
 		} else if podListLen < 1 {
+			// Do not report LocalImageURL while deploying
+			instance.Status.LocalImageURL = ""
```

Comment:
Still not quite right, gets cleared for a seconds but gets set as soon as the pod exists but is not ready to serve